src/components/form/dt-location-map/dt-location-map-item.js

```diff
@@ -162,32 +162,45 @@ export default class DtLocationMapItem extends LitElement {
         }
 
         .field-container {
+          --dt-location-map-border-radius: var(--dt-form-border-radius, 10px);
```

Comment:
Can we put the default/fallback for this as 0? None of the other form fields have a radius by default, so this should be the same, I think.

Reply:
oops, yup. Put back to rights. I meant to undo that before pushing.